README.md

```diff
@@ -38,7 +38,7 @@ and multiple formats,
 including
 1. Java, .NET Core, NodeJS, PHP, and Python auto-instrument agents.
 1. Go and C++ SDKs.
-1. LUA agent especially for Nginx, OpenResty.
+1. LUA agent especially for Nginx, OpenResty and [Apache APISIX](https://github.com/apache/apisix).
```

Comment:
Readme usually doesn't provide a link. Please remove it and keep `Apache APISIX` only.

Reply:
fixed